andela/travel_tool_front
# #162474402 Restore yarn lock to pr 348 version

#### What does this PR do?
- restore the previous version of the yarn lock file so that the date pickers work.

#### Description of Task to be completed?
- restore the previous version of the yarn lock file so that the date pickers work.

#### How should this be manually tested?
- Setting up the backend
     - Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
     - Enter into the project directory - `cd travel_tool_back`
     - checkout the branch git checkout `develop`
     - start the backend server with `make start`


- Setting up the frontend
   - Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
   - Enter into the project directory - `cd travel_tool_front`
   - Checkout to the branch `ch-restore-yarnlock-348` then run the app `make start`
   - Login to the application
   - Navigate to the requests and attempt to create a travel request
   - It is expected that the date pickers will update the input fields when a date is picked.

#### Any background context you want to provide?
- Should the date pickers fail to update, try running `make ssh` and run yarn install.
<img width="617" alt="screenshot 2018-12-13 at 09 53 23" src="https://user-images.githubusercontent.com/21138053/49920926-f4d97980-febc-11e8-8f7b-198ca98f8434.png">


#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A
